<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminErrorReports</class>
 <widget class="QWidget" name="AdminErrorReports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>671</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="lbl_stripedBackground">
     <property name="geometry">
      <rect>
       <x>-150</x>
       <y>-70</y>
       <width>671</width>
       <height>841</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/images/stripedBackground.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pageLeft">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>580</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/images/pageLeft.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_report_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:24px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pageRight">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>580</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/pageRight.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_report_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:24px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view_4">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>300</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  rgba(100,100,100,0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_report_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:24px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_currentPage">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>590</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font-family:arial;
font-size:18px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view_2">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>140</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  rgba(100,100,100,0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_report_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:24px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_report_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:24px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view_5">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>390</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  rgba(100,100,100,0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_topBar">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>-15</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/topBar.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_backToAdminHome">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/backArrow2.png);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_report_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>510</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:24px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view_6">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>470</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  rgba(100,100,100,0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_reportedIssues">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font-size:28px;
font-family:arial;</string>
     </property>
     <property name="text">
      <string>    Reported Issues</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view_3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>220</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  rgba(100,100,100,0);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_view_1">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>60</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  rgba(255,255,255,0.1);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_name_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_name_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_name_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_name_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_name_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_name_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>480</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_date_1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;
text-align: right;</string>
     </property>
     <property name="text">
      <string>24/06</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_date_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;
text-align: right;</string>
     </property>
     <property name="text">
      <string>24/06</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_date_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;
text-align: right;</string>
     </property>
     <property name="text">
      <string>24/06</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_date_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>310</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;
text-align: right;</string>
     </property>
     <property name="text">
      <string>24/06</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_date_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>400</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;
text-align: right;</string>
     </property>
     <property name="text">
      <string>24/06</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_date_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>480</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: arial;
font-size: 14px;
color: black;
text-align: right;</string>
     </property>
     <property name="text">
      <string>24/06</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_new_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/newMarker.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_new_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/newMarker.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_new_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/newMarker.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_new_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>340</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/newMarker.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_new_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>430</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/newMarker.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_new_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>510</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/newMarker.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>lbl_stripedBackground</zorder>
    <zorder>lbl_report_2</zorder>
    <zorder>btn_pageLeft</zorder>
    <zorder>lbl_report_1</zorder>
    <zorder>btn_pageRight</zorder>
    <zorder>lbl_report_4</zorder>
    <zorder>lbl_report_5</zorder>
    <zorder>lbl_currentPage</zorder>
    <zorder>lbl_report_3</zorder>
    <zorder>lbl_topBar</zorder>
    <zorder>btn_backToAdminHome</zorder>
    <zorder>lbl_report_6</zorder>
    <zorder>lbl_reportedIssues</zorder>
    <zorder>lbl_name_1</zorder>
    <zorder>lbl_name_2</zorder>
    <zorder>lbl_name_3</zorder>
    <zorder>lbl_name_4</zorder>
    <zorder>lbl_name_5</zorder>
    <zorder>lbl_name_6</zorder>
    <zorder>lbl_date_1</zorder>
    <zorder>lbl_date_2</zorder>
    <zorder>lbl_date_3</zorder>
    <zorder>lbl_date_4</zorder>
    <zorder>lbl_date_5</zorder>
    <zorder>lbl_date_6</zorder>
    <zorder>lbl_new_1</zorder>
    <zorder>lbl_new_2</zorder>
    <zorder>lbl_new_3</zorder>
    <zorder>lbl_new_4</zorder>
    <zorder>lbl_new_5</zorder>
    <zorder>lbl_new_6</zorder>
    <zorder>btn_view_5</zorder>
    <zorder>btn_view_6</zorder>
    <zorder>btn_view_3</zorder>
    <zorder>btn_view_2</zorder>
    <zorder>btn_view_1</zorder>
    <zorder>btn_view_4</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="lbl_background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>671</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_viewReport">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>-5</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font-size:28px;
font-family:arial;</string>
     </property>
     <property name="text">
      <string>    View Report</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_topBar_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/topBar.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_backToReportList">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>-5</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/backArrow2.png);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_reportTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:18px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_sender">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:14px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_previousReport">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/images/pageLeft.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_nextReport">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/pageRight.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_currentReport">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:14px;
color:black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
font-size:14px;
color: black;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_errorText">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;
color:gray;
font-size:18px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_markAsRead">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>620</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family:arial;</string>
     </property>
     <property name="text">
      <string>Mark As Read</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_newMarker">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/images/newMarker.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>lbl_background</zorder>
    <zorder>lbl_topBar_2</zorder>
    <zorder>btn_backToReportList</zorder>
    <zorder>lbl_viewReport</zorder>
    <zorder>lbl_reportTitle</zorder>
    <zorder>lbl_sender</zorder>
    <zorder>btn_previousReport</zorder>
    <zorder>btn_nextReport</zorder>
    <zorder>lbl_currentReport</zorder>
    <zorder>lbl_date</zorder>
    <zorder>lbl_errorText</zorder>
    <zorder>btn_markAsRead</zorder>
    <zorder>lbl_newMarker</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
